### ISSUE TITLE
Why so complicated?

### ISSUE THREAD

**Issue by GabrielDornelles** [mask-classifier/mask-classifier#1]
Hey, I just saw your post on linkedin and I completely lost it, anyway, I feel like I could contribute or at least question about.
I REALLY like to see this kind of work so dont misunderstand that like pure critisicm, I really liked your work!
You loaded the MobileNet model with
```py
baseModel = MobileNet(weights="imagenet", include_top=False,input_tensor=Input(shape=(224, 224, 3)))
```
I use PyTorch, so Im not completely familiar with tensorflow or keras syntax, but I feel like the include top only means that you want to build with a different type of input tensor, which in your case is 224 224 3, same as every net since ImageNet uses this shape. In this cases theres no reason to change that.
But even considering that you wanted by some reason change this layer like you did
```py
headModel = baseModel.output
headModel = AveragePooling2D(pool_size=(7, 7))(headModel)
headModel = Flatten(name="flatten")(headModel)
headModel = Dense(128, activation="relu")(headModel)
headModel = Dropout(0.5)(headModel)
headModel = Dense(2, activation="softmax")(headModel)
```
it makes no sense at all because MobileNet is already optimized and its architecture runs the following:
![image](https://user-images.githubusercontent.com/56324869/117449734-b07c2780-af16-11eb-8470-c4ba0991b4d6.png)

So (I think) you guys changed in order to change the last last layer FC from 1000 neurons that the original has for ImageNet to 2 (using mask or not using mask), in this case you have a parameter called classes for that
```py
tf.keras.applications.MobileNet(
    input_shape=None, alpha=1.0, depth_multiplier=1, dropout=0.001,
    include_top=True, weights='imagenet', input_tensor=None, pooling=None,
    classes=1000, classifier_activation='softmax', **kwargs
)
```
Since its a binary problem a sigmoid and a single neuron on the last layer could solve as well, but I assume you guys tried but accuracy was better for softmax on 2 neurons. Also, I highly recommend to try some really simple net, binary problems usually dont require much processment and MobileNet is optimized for mobile because of its architecture using inverted bottlenecks(Mobile Inverted bottleneck convolution, or Mbconv in papers), but again, binary classification can most of the time be done with really simple nets that dont need all of that stuff, MobileNet runs that for 1000 classes, for 2 classes your model could be much much smaller.
Cheers.
